tests/INTEGRATION-TESTS.md

```diff
@@ -34,7 +34,7 @@ IronOxide tests can be run against any other environment, with proper setup. To
 
 ### Identity Assertion Key File
 
-An Identity Assertion Key file must be downloaded from the admin console interface immediately after creating a new Identity Assertion Key. It must be named `iak.pem` and placed in `./tests/testkeys/`.
+An Identity Assertion Key file must be downloaded from the admin console interface immediately after creating a new Identity Assertion Key. It must be named `iak-${IRONCORE_ENV}.pem` (defaults to `iak-prod.pem` if the IRONCORE_ENV variable is not set) and placed in `./tests/testkeys/`.
```

Comment:
This isn't accurate. This markdown section is for testing against a different environment (not dev, stage, or prod), when they choose to set `IRONCORE_ENV=htttps://some-url.com`. In this case, the file must be named exactly `iak.pem`. 

It's true that it defaults to prod when the variable is unset, but that shouldn't go into the `Testing against a different environment` section if we want to add that in. And since there are default configs/keys encrypted for IronCore use, I personally don't think that we need to call this out.

Reply:
I see - I jumped into the "other environments" section and thought the IAK always needed to be in `iak.pem`. I'll revert this.